roxctl/analyze/netpol/cta.go

```diff
@@ -46,7 +49,11 @@ func (cmd *analyzeNetpolCommand) ouputConnList(connsStr string) error {
 	if cmd.outputToFile {
 		dirpath, filename := filepath.Split(cmd.outputFilePath)
 		if filename == "" {
-			filename = defaultOutputFileName
+			fileSuffix := cmd.outputFormat
+			if cmd.outputFormat == "" {
+				fileSuffix = "txt"
+			}
+			filename = defaultOutputFileNamePrefix + fileSuffix
```

Comment:
why split `cmd.outputFilePath` ? I assume if the arg is empty it should consider the default.

Reply:
I assumed the same too, but after updating the `netpol_test.go` (line 223 ), the relevant test failed (the expected file (`connlist.txt`) didn't exist, but a file without the suffix (`connlist.`) existed). 
so I added a debug print here and found that `cmd.outputFormat` was empty .(still could not understand why)

however the default output is txt anyway because it is defined well in netpol-analyzer's getFormatter func

Reply:
my fault ..... I had to update default value in netpol_test.go flags too
this is reverted

Reply:
yes this is true, and works correctly when running from command line.(the cmd.outputFormat takes the default value if we don't use the flag) and also works correctly for bats tests.

BUT, fails when running the unit tests (started to fail after adding the the .txt suffix in netpol_test.go line 123, 125)
--------------- so I will revert theses change , but then we'll have the following bug: 
the unit test : `"output should be written to default output file"`
will generate default file with the name connlist. instead of connlist.txt 

the default value of the flag is not considered when we run unit tests , since although we defined a `testCmd ` with the flags, we didn't actually use it.
`testCmd `  is not sent to the `analyzeNetpolCmd.construct` .

(comparing analyze/netpol_test.go with generate/netpol_test.go -> in generate `testCmd `  is sent to the `analyzeNetpolCmd.construct`  and handled with cmd.splitMode ... )

Reply:
fixed the bug in `netpol_test.go` by assigning relevant default values  and eliminating the unused `testCmd `